plugins/modules/pnp_intent.py

```diff
@@ -728,13 +753,16 @@ class instance for further use.
                     function='get_device_list',
                     params={"serial_number": device["deviceInfo"]["serialNumber"]}
                 )
-
+                self.log("Device details for the serial number {0}\
```

Comment:
self.log("Device details for serial number {0} obtained from the API 'get_device_list': {1}".format(device["deviceInfo"]["serialNumber"], str(multi_device_response)), "DEBUG")

Reply:
Resolved